docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
 title: DUNE
 ---
 
-[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool that allows blockchain developers and node operators to perform boilerplate tasks related to smart contract development and node management functions.
+[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool. 
+As a blockchain developer or node operator, use DUNE for your common tasks.
 
-Before getting started with smart contract development, you need to learn about DUNE and how to install it on your platform.
+Before you begin, learn about DUNE and how to install it on your platform.
```

Comment:
This changes the meaning. 

The intention here seemed to be "before you begin **smart contract development**.
Now it is ambiguous

Reply:
"Before getting started ..." is Grade 12 level. The new construction is Grade 6.

Reply:
Done